controllers/restic.go

```diff
@@ -171,8 +171,11 @@ func (r *DPAReconciler) buildResticDaemonset(dpa *oadpv1alpha1.DataProtectionApp
 		return nil, fmt.Errorf("ds cannot be nil")
 	}
 
+	// get resource requirements for restic ds
+	resticResourceReqs, _ := r.getResticResourceReqs(dpa)
```

Comment:
Ideally, this error should be surfaced into the DPA CR so that users are aware of the invalid values they put

Reply:
I would add a comment about err check that is done in validator.go

Reply:
Done